Packs/qualys/Integrations/QualysEventCollector/QualysEventCollector.yml

```diff
@@ -0,0 +1,141 @@
+category: Vulnerability Management
+sectionOrder:
+- Connect
+- Collect
+commonfields:
+  id: QualysEventCollector
+  version: -1
+configuration:
+- defaultvalue: https://qualysguard.qg2.apps.qualys.com
+  display: Server URL
+  name: url
+  required: true
+  type: 0
+  section: Connect
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+  section: Connect
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+  section: Connect
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+  section: Connect
+- defaultvalue: 3 days
+  section: Collect
+  display: First fetch time
+  name: first_fetch
+  additionalinfo: If "First Fetch Time" is set for a long time ago, it may cause performance issues.
+  required: true
+  type: 0
+- display: Vulnerability Fetch Interval
+  additionalinfo: Time between fetches of vulnerabilities (for example 12 hours, 60 minutes, etc.).
+  name: host_detections_fetch_interval
+  required: true
+  section: Collect
+  defaultvalue: 12 hours
+  type: 0
+- section: Collect
+  advanced: true
+  display: Activity Logs Fetch Interval
+  additionalinfo: Time between fetches of activity logs.
+  name: eventFetchInterval
+  defaultvalue: "1"
+  type: 19
+  required: false
+- defaultvalue: 10000
+  section: Collect
+  display: Activity Logs Fetch Limit
+  name: max_fetch_activity_logs
+  additionalinfo: Maximum number of activity logs to fetch per fetch iteration.
+  required: true
+  type: 0
+- defaultvalue: 1000
+  section: Collect
+  display: Host Detections Fetch Limit
+  name: max_fetch_hosts_detections
+  additionalinfo: Maximum number of hosts to return in a single fetch iteration.
```

Comment:
```suggestion
  additionalinfo: Maximum number of hosts to return in a single fetch iteration of host detections. Since each host may have multiple detections, it is likely that more events than the specified number will be fetched.
```